dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -626,6 +626,13 @@ internal class CoreFeature(
         } else {
             appContext.packageName == currentProcess.processName
         }
+        if (!isMainProcess) {
+            internalLogger.log(
```

Comment:
@reviewers should we also send a telemetry here ? I think it is a bit overkill, what do you think ?

Reply:
would it be actionable? how it would help us?

Reply:
not sure it will be that actionable, that is the reason why I did not add it and just asked your opinion. I think we are better off.

Reply:
I think it's an interesting data to know at least, and if we notice that there are really amount of clients doing it in background (probably meaning intentionally), the action will be making it work even if the initialization happens in background.